### DISCUSSION TITLE
Can the DatePicker display the calendar popover when the date segments gain focus?

### DISCUSSION THREAD

**Discussion by will-stone** [adobe/react-spectrum#6970] [Q&A]
Hi 👋🙂

Using the RAC `DatePicker`, is it possible to display the calendar as soon as the input segments gain focus? I've tried using state and setting `isOpen` on focus but the popover steals focus and the segments are no longer interactive. We're trying to build something similar to how IBM Carbon does it with their picker: https://carbondesignsystem.com/components/date-picker/usage/

Thank you.

**Comment by reidbarber** [comment]
Seems like a reasonable feature request. We have a similar option in Combobox: https://react-spectrum.adobe.com/react-aria/ComboBox.html#menu-trigger-behavior

Feel free to convert this into an issue for further discussion.

**Reply by will-stone** [reply]
Thanks! Will do...
